manageiq-operator/pkg/apis/manageiq/v1alpha1/manageiq_types.go

```diff
@@ -4,12 +4,14 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
```

Comment:
Please remove the boiler plate comments.

Reply:
Done